psi4/src/psi4/libmints/bessel.cc

```diff
@@ -172,7 +173,7 @@ void BesselFunction::calculate(const double z, int maxL, std::vector<double> &va
 		} else {
 			// Determine the necessary derivatives from
 			// K_l^(n+1) = C_l K_(l-1)^(n) + (C_l + 1/(2l+1))K_(l+1)^(n) - K_l^(n)
-			double dK[TAYLOR_CUT+1][maxLambda];
+			std::vector<std::vector<double>> dK(TAYLOR_CUT+1, std::vector<double>(maxLambda));
```

Comment:
I would like to see this allocation along with the `dzn` below happen outside of the `calculate` function which is called many many times. `maxLambda` is `maxL + TAYLOR_CUT`. `maxL` might be replaced with `MAX_AM_ERI` (if I'm back tracking the code probably.